generator/fluentReportsGenerator.js

```diff
@@ -6196,8 +6343,19 @@ class UI { // jshint ignore:line
     }
 
     _handleShowProperties(props, obj, table, layout) {
+        let propertyToSkip = -1;
+        if(obj.text !== undefined || obj.function !== undefined){
+            let newObj = shallowClone(obj);
+            delete newObj.formatFunction;
+            for(let i=0;i<newObj.properties.length;i++){
```

Comment:
When I try to add a new "Print label", I get a
> Cannot read property 'length' of undefined

On this line.

Reply:
Same as above, I should have ran the tests and caught this before I sent it on to you...